src/themes/OLH/templates/journal/article.html

```diff
@@ -104,36 +104,36 @@ <h3>{{ article.title|safe }}</h3>
             <div class="mini-bar sticky" data-sticky data-margin-top="0" data-sticky data-anchor="content">
                 <div class="row">
                     <div class="title-bar" data-responsive-toggle="options-menu" data-hide-for="medium">
-                        <button class="menu-icon" type="button" data-toggle></button>
+                        <button class="menu-icon" type="button" data-toggle aria-label="Options"></button>
                         <div class="title-bar-title">{% trans 'Options' %}</div>
                     </div>
                     <div id="options-menu">
-                        <div class="large-6 columns">
+                        <div class="large-3 medium-4 columns">
                             <ul class="menu vertical medium-horizontal pad-left-15"
                                 data-responsive-menu="drilldown medium-dropdown">
                               {% with article.get_doi_url as doi_url %}
                                 <li>{% trans "Share" %}:</li>
-                                <li><a href="https://www.facebook.com/share.php?p[url]={% if doi_url %}{{ doi_url }}{% else %}{{ article.url }}{% endif %}" target="_blank"><i class="fa fa-facebook"></i></a></li>
-                                <li><a href="https://twitter.com/intent/tweet?text={{ article.title }} {% if doi_url %}{{ doi_url }}{% else %}{{ article.url }}{% endif %}" target="_blank"><i class="fa fa-twitter"></i></a></li>
-                                <li><a href="https://www.linkedin.com/sharing/share-offsite?url={% if doi_url %}{{ doi_url }}{% else %}{{ article.url }}{% endif %}" target="_blank"><i class="fa fa-linkedin"></i></a></li>
+                                <li><a href="https://www.facebook.com/share.php?p[url]={% if doi_url %}{{ doi_url }}{% else %}{{ article.url }}{% endif %}" target="_blank" aria-label="Share on Facebook"><i class="fa fa-facebook"></i></a></li>
+                                <li><a href="https://twitter.com/intent/tweet?text={{ article.title }} {% if doi_url %}{{ doi_url }}{% else %}{{ article.url }}{% endif %}" target="_blank" aria-label="Share on X"><i class="fa fa-twitter"></i></a></li>
+                                <li><a href="https://www.linkedin.com/sharing/share-offsite?url={% if doi_url %}{{ doi_url }}{% else %}{{ article.url }}{% endif %}" target="_blank" aria-label="Share on Linked In"><i class="fa fa-linkedin"></i></a></li>
                               {% endwith %}
                             </ul>
                         </div>
-                        <div class="large-6 columns">
-                            <ul class="menu vertical medium-horizontal align-right"
+                        <div class="large-4 medium-5 columns">
```

Comment:
unless I'm missing something, the values no longer add up to 12 columns on these divs (`large-3` + `large-4`). If we want to leave a gap I'd suggest adding another div (`<div class="large-5 medium-3"`)

Reply:
I did look into that - both in the framework docs and then it took a bit of trial and error to find values that worked when tested.  As I understood it values didn't need to add up to 12 - rather they set the minimum space, and when the sum was less than twelve the remaining space was managed dynamically so that the it balanced as the window resized.

Since there are the other changes to make I'll look at this again too, but from memory it wasn't needed.

Reply:
> Incomplete Rows
In order to work around browsers' different rounding behaviors, Foundation will float the last column in a row to the right so the edge aligns. If your row doesn't have a count that adds up to 12 columns, you can tag the last column with a class of end in order to override that behavior.

https://get.foundation/sites/docs-v5/components/grid.html#:~:text=You%20can%20use%20column%20or,devices%20will%20inherit%20those%20styles.

Reply:
I think they describe it further down:
![image](https://github.com/BirkbeckCTP/janeway/assets/10760678/2419be99-3325-4e46-8e8b-fa8a13649582)


Since the framework will float the last column, the default behaviour will be up to the browser implementation. On Firefox, I have seen gaps whenever columns don't add up to 12. So I believe it is worth ensuring that here, even if just for consistency.

Reply:
ok